lib/active_support_type_extensions/array.rb

```diff
@@ -9,7 +9,7 @@ def serializable?(_)
     def cast_value(value)
       case value
       when String
-        value.split(",")
+        cast_value(value.split(/,|\|/))
```

Comment:
This technically also allows parsing strings like `a,b|c`. Should we allow mixed separators?

Reply:
It’s probably something that won’t happen that often, but I’ve found a way of preventing the mixing of separators. It will hurt your brain, though :joy: 
```ruby
value.split(/,(?!.*\|)|\|(?!.*,)/)
```